src/utils/formik/index.js

```diff
@@ -30,20 +30,20 @@ const wrapper = (Component, options) => ({ field, form, ...properties }) => {
     hasIsFieldProp: false,
     ...options,
   };
-  const executeDebounced = (() => {
+
+  const [executeDebounced] = useState(() => {
```

Comment:
what's the reasoning for placing this in the state?

Reply:
I needed the `executeDebounced` function to be the same with each render, so I used `useState` for that.
The problem was that with each new render a new `executeDebounced` function was created so the value of the input was updated in a race condition type of situation.

Reply:
or even put it outside the component and just pass the debounce as parameter when used

Reply:
It needs to be unique for each component. If it were to live outside it would be unique across components which I don't think it would work properly.
`useCallback` should work too, same with `useMemo` and others. `useState` just seemed more appropriate at the moment of writing the code. If you prefer I can change it to use `useCallback` instead. I personally don't have a strong opinion towards one or the other.

Reply:
@ifrim semantically I find that more fit for this kind of scenarios